anime_downloader/sites/animechameleon.py

```diff
@@ -0,0 +1,30 @@
+from anime_downloader.sites import helpers
+from anime_downloader.sites.anime import Anime, AnimeEpisode, SearchResult
+
+class AnimeChameleon(Anime, sitename = 'gurminder'):
+    sitename = "gurminderboparai"
+    @classmethod
+    def search(cls, query):
+        soup = helpers.soupify(helpers.get('http://anime.gurminderboparai.com/search/{}'.format(query)).text).find('div', class_='panel-body').find_all('a')
+        search_results = [
+            SearchResult(
+                title = x.text,
+                url = x['href']
+                )
+            for x in soup
+            ]
+        return search_results
+
+    def _scrape_episodes(self):
+        soup = helpers.soupify(helpers.get(self.url).text).find('ul', id='episodes-list').find_all('li')
+        eps = [x.a['href'] for x in soup]
+        eps.reverse()
+        return eps
+
+    def _scrape_metadata(self):
+        self.title = helpers.soupify(helpers.get(self.url).text).find('h3', class_='panel-title').text
+
+class AnimeChameleonEpisode(AnimeEpisode, sitename='gurminder'):
```

Comment:
It errors here I'm pretty sure:
```
2020-07-26 16:17:31 arch anime_downloader.sites.anime[4844] DEBUG EPISODE IDS: length: 12, ids: ['http://anime.gurminderboparai.com/anime/One%20Punch%20Man/1', 'http://anime.gurminderboparai.com/anime/One%20Punch%20Man/2', 'http://anime.gurminderboparai.com/anime/One%20Punch%20Man/3', 'http://anime.gurminderboparai.com/anime/One%20Punch%20Man/4', 'http://anime.gurminderboparai.com/anime/One%20Punch%20Man/5', 'http://anime.gurminderboparai.com/anime/One%20Punch%20Man/6', 'http://anime.gurminderboparai.com/anime/One%20Punch%20Man/7', 'http://anime.gurminderboparai.com/anime/One%20Punch%20Man/8', 'http://anime.gurminderboparai.com/anime/One%20Punch%20Man/9', 'http://anime.gurminderboparai.com/anime/One%20Punch%20Man/10', 'http://anime.gurminderboparai.com/anime/One%20Punch%20Man/11', 'http://anime.gurminderboparai.com/anime/One%20Punch%20Man/12']                                                         
2020-07-26 16:17:31 arch anime_downloader.commands.dl[4844] INFO Found anime: One Punch Man
Traceback (most recent call last):
  File "/tmp/anime_downloader_patch-8_b95cb6b4-cf4a-11ea-9176-90324b7a94df/bin/anime", line 11, in <module>
    load_entry_point('anime-downloader==4.4.0', 'console_scripts', 'anime')()
  File "/tmp/anime_downloader_patch-8_b95cb6b4-cf4a-11ea-9176-90324b7a94df/lib/python3.8/site-packages/anime_downloader-4.4.0-py3.8.egg/anime_downloader/cli.py", line 53, in main
    cli()
  File "/tmp/anime_downloader_patch-8_b95cb6b4-cf4a-11ea-9176-90324b7a94df/lib/python3.8/site-packages/click-7.1.2-py3.8.egg/click/core.py", line 829, in __call__
    return self.main(*args, **kwargs)
  File "/tmp/anime_downloader_patch-8_b95cb6b4-cf4a-11ea-9176-90324b7a94df/lib/python3.8/site-packages/click-7.1.2-py3.8.egg/click/core.py", line 782, in main
    rv = self.invoke(ctx)
  File "/tmp/anime_downloader_patch-8_b95cb6b4-cf4a-11ea-9176-90324b7a94df/lib/python3.8/site-packages/click-7.1.2-py3.8.egg/click/core.py", line 1259, in invoke
    return _process_result(sub_ctx.command.invoke(sub_ctx))
  File "/tmp/anime_downloader_patch-8_b95cb6b4-cf4a-11ea-9176-90324b7a94df/lib/python3.8/site-packages/click-7.1.2-py3.8.egg/click/core.py", line 1066, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/tmp/anime_downloader_patch-8_b95cb6b4-cf4a-11ea-9176-90324b7a94df/lib/python3.8/site-packages/click-7.1.2-py3.8.egg/click/core.py", line 610, in invoke
    return callback(*args, **kwargs)
  File "/tmp/anime_downloader_patch-8_b95cb6b4-cf4a-11ea-9176-90324b7a94df/lib/python3.8/site-packages/click-7.1.2-py3.8.egg/click/decorators.py", line 21, in new_func
    return f(get_current_context(), *args, **kwargs)
  File "/tmp/anime_downloader_patch-8_b95cb6b4-cf4a-11ea-9176-90324b7a94df/lib/python3.8/site-packages/anime_downloader-4.4.0-py3.8.egg/anime_downloader/commands/dl.py", line 99, in command
    animes = util.parse_ep_str(anime, episode_range)
  File "/tmp/anime_downloader_patch-8_b95cb6b4-cf4a-11ea-9176-90324b7a94df/lib/python3.8/site-packages/anime_downloader-4.4.0-py3.8.egg/anime_downloader/util.py", line 137, in parse_ep_str
    return split_anime(anime, parse_episode_range(anime, grammar))
  File "/tmp/anime_downloader_patch-8_b95cb6b4-cf4a-11ea-9176-90324b7a94df/lib/python3.8/site-packages/anime_downloader-4.4.0-py3.8.egg/anime_downloader/util.py", line 115, in split_anime
    anime = anime[start-1:end-1]
  File "/tmp/anime_downloader_patch-8_b95cb6b4-cf4a-11ea-9176-90324b7a94df/lib/python3.8/site-packages/anime_downloader-4.4.0-py3.8.egg/anime_downloader/sites/anime.py", line 178, in __getitem__
    episode_class = AnimeEpisode.subclasses[self.sitename]
KeyError: 'gurminderboparai'
```
No idea why it's still using gurminderboparai on a clean clone though